### ISSUE TITLE
Are scripts compatible with Blender 2.8 ?

### ISSUE THREAD

**Issue by xan2622** [Stromberg90/Scripts#1]
Are scripts compatible with Blender 2.8 ?

I keep trying to install them but they never get listed in the addons list.
Am I doing something wrong or maybe the scripts are not compatible with blender 2.8 ?

Thanks for your help.

**Comment by Stromberg90** [comment]
ContextSelect and MergeTool are compatible, the others are not yet.
Which ones did you try?

**Comment by xan2622** [comment]
All of them but I mostly would like to use MergeTool and ContextSelect.

I do click on the [Install...] button and select the MergeTool.py file but after this step, when I search for "MergeTool" or "Merge" in the Preferenecs > Addons window, I find nothing.

**Comment by Stromberg90** [comment]
![blender_2019-06-11_01-38-35](https://user-images.githubusercontent.com/9784913/59233657-a7c72d80-8be9-11e9-8d84-f2913210763b.png)

I reproduced your steps, it does show up for me.
Could you go to Window->Toggle Systems Console and see if any errors show up when you try to install it?

**Comment by xan2622** [comment]
I have just downloaded the latest (Windows 64 bits) nightly build from : https://builder.blender.org/download/

Here is what the Systems Console shows:

<img width="1044" alt="screenshot_2019-06-11_080738" src="https://user-images.githubusercontent.com/1732773/59247595-263ec200-8c20-11e9-80ab-7a230dda1754.png">

**Comment by Stromberg90** [comment]
I can't reproduce this on my end, I updated ContextSelect and MergeTool 4 months ago, could you be using older files than that?

**Comment by xan2622** [comment]
I just download the ones from github, here, from your repository.
I have just retried a few minutes ago (with another fresh nightly build). Same errors as the one in the Systems Console (above screenshot).

To install them, do you click on the Install button (in Preferences > Addons)? 

Or do you copy the .py files manually and paste them into some folder?
C:\Users\username\AppData\Roaming\Blender Foundation\Blender\2.80\scripts\addons\ for example ? 

I only click on the Install button.
And here is what I see as soon as I click on the button: 
![ACxG6Ej6LU](https://user-images.githubusercontent.com/1732773/59460874-64a2cf80-8e20-11e9-88ff-1c2702f1bec0.jpg)
I checked in that folder, the .py file is there (correctly copied).

FrankIy don't understand what is happening.

Could it be related to:
https://blender.stackexchange.com/questions/48085/how-to-fix-fake-module-addon-missing-bl-info-give-bad-performance/64497

**Comment by Stromberg90** [comment]
I don't understand either, can you try deleting my other scripts in that folder, and only install the mergetool as a start?
I did it by using the install button in blender when I tested.

**Comment by xan2622** [comment]
I have asked for advices and help on a blender Discord server and here is what someone showed me:
![error](https://i.ibb.co/6wJ4QLK/unknown-1.png)

**Comment by xan2622** [comment]
As I continue to investigate about this problem, this morning I have completely deleted my 
**C:\Users\Nadine\AppData\Roaming\Blender Foundation\** folder and relaunched blender (starting with a default configuration), it didn't help. Same problem.

Tested with the latest build *blender-2.80.0-git.8ac0fef4dfd1-windows64*

**Comment by xan2622** [comment]
I have been advised to try with a portable version of blender if the problem persists. 
It does.

**Comment by xan2622** [comment]
**I found the reason why !** 

When I downloaded your scripts from https://github.com/Stromberg90/Scripts/tree/master/Blender, I used to right click on the links (over the scripts links) and click on "Save target as..." in Firefox. 

But if this method did download the files (and their extension) correctly, it saved bad files: inside the files, their content were not python but HTML !

<img width="631" alt="screenshot2019-06-14_094149" src="https://user-images.githubusercontent.com/1732773/59492217-c8f97980-8e88-11e9-83be-71db5a8e075e.png">

It seems that the correct way to download files from github is to open the files on github and save them as RAW files.

Sorry for the inconvenience.
